src/app/components/PodcastCard.tsx

```diff
@@ -0,0 +1,122 @@
+import { PostPodcast } from '@/app/lib/posts/types'
+import Image from 'next/image'
+import { motion } from 'framer-motion'
+import { Skeleton } from '@/app/components/ui/skeleton'
+import Link from 'next/link'
+
+type PodcastCardProps = {
+  post?: PostPodcast
+  isLoading?: boolean
+  newPost?: boolean
+}
+
+export default function PodcastCardProps({
+  post,
+  isLoading = false,
+  newPost = false,
+}: PodcastCardProps) {
+  if (isLoading) {
+    return <PocastCardSkeleton />
+  }
+
+  if (!post) return null
+
+  return (
+    <motion.div
+      whileHover={{ x: -2, y: -4, boxShadow: '8px 8px 0px #000' }}
+      whileTap={{ x: -1, y: -1, boxShadow: '2px 2px 0px #000' }}
+      transition={{ duration: 0.12 }}
+      className="outline outline-[1px] outline-primary rounded-lg bg-white hover:outline-[3px] overflow-hidden"
+    >
+      <div className="flex flex-row items-center justify-center p-10 w-full h-auto">
+        <div className="flex flex-col sm:flex-row lg:flex-col xl:flex-row justify-center items-center w-full gap-8 h-full">
+          {post.thumbnail_url && (
+            <div className="h-auto w-full xl:h-[225px] min-w-[225px] lg:w-full">
+              <Image
+                src={
+                  post.thumbnail_url !== 'string'
+                    ? post.thumbnail_url
+                    : 'https://tinyurl.com/25ks83w3'
+                }
+                alt={post.title}
+                width={225}
+                height={225}
+                sizes="100vw"
+                className="border border-primary border-opacity-10 rounded-xl h-full w-full xl:w-auto"
+              />
+            </div>
+          )}
+          <div className="flex flex-col w-full justify-between gap-5 h-[225px]">
+            <div className="flex flex-col justify-start items-start h-fit gap-1">
+              <div className="flex justify-center items-center gap-2">
+                <p className="font-mono text-base line-clamp-1  ">
+                  {post.duration}
+                  {' • '}
+                  {post.publish_date}
+                </p>
+                {newPost && (
+                  <span className="font-bold-condensed text-white bg-blue rounded-full px-3 py-0.5 text-sm uppercase tracking-wide">
+                    New
+                  </span>
+                )}
+              </div>
+              <h3 className="text-xl font-bold-condensed uppercase -tracking-[0.050rem] line-clamp-2 md:line-clamp-3">
+                {post.title!.length > 160
+                  ? post.title!.substring(0, 160) + '...'
+                  : post.title}
+              </h3>
+              <p className="line-clamp-2 text-lg leading-[24px]">
+                {post.meta_default_description!.length > 160
+                  ? post.meta_default_description!.substring(0, 160) + '...'
```

Comment:
Do we need this?

Won't **line-clamp-2** do the job?

I think I've use that logic for the post description, but without line-clamp-2

I'd use line-clamp-2 only tho (I will apply that to post later)